govc/vm/disk/create.go

```diff
@@ -61,6 +62,7 @@ func (cmd *create) Register(ctx context.Context, f *flag.FlagSet) {
 	f.Var(&cmd.Bytes, "size", "Size of new disk")
 	f.BoolVar(&cmd.Thick, "thick", false, "Thick provision new disk")
 	f.BoolVar(&cmd.Eager, "eager", false, "Eagerly scrub new disk")
+	f.StringVar(&cmd.DiskMode, "mode", "dependent", "One of: dependent (also known as persistent), persistent, nonpersistent, undoable, independent_persistent, independent_nonpersistent, append")
```

Comment:
:+1: on adding the option.  Could you change the usage similar to the other choice options?  Some of the examples:
```
govc/datastore/create.go:119:	f.Var(&cmd.Type, "type", fmt.Sprintf("Datastore type (%s)", strings.Join(allTypes, "|")))
govc/datastore/create.go:126:		fmt.Sprintf("Access mode for the mount point (%s)", strings.Join(modes, "|")))
govc/device/scsi/add.go:53:		fmt.Sprintf("SCSI controller type (%s)", strings.Join(ctypes, "|")))
govc/dvs/portgroup/add.go:57:		fmt.Sprintf("Portgroup type (%s)", strings.Join(ptypes, "|")))
```